lib/src/local_working_copy.rs

```diff
@@ -1617,14 +1680,23 @@ impl FileSnapshotter<'_> {
         path: &RepoPath,
         disk_path: &Path,
     ) -> Result<FileId, SnapshotError> {
-        let file = File::open(disk_path).map_err(|err| SnapshotError::Other {
-            message: format!("Failed to open file {}", disk_path.display()),
-            err: err.into(),
-        })?;
-        Ok(self
-            .store()
-            .write_file(path, &mut BlockingAsyncReader::new(file))
-            .await?)
+        let file =
+            File::options()
+                .read(true)
+                .open(disk_path)
+                .map_err(|err| SnapshotError::Other {
+                    message: format!("failed to open the file {}", disk_path.display()),
```

Comment:
nit: "Failed to open" (I don't know if "the" should be added, btw.)

Reply:
Done.